xml/templates/message_templates/participant_transferred_html.tpl

```diff
@@ -34,58 +34,85 @@
      </tr>
      <tr>
       <td colspan="2" {$valueStyle}>
-       {$event.event_title}<br />
-       {$event.event_start_date|crmDate}{if $event.event_end_date}-{if $event.event_end_date|crmDate:"%Y%m%d" == $event.event_start_date|crmDate:"%Y%m%d"}{$event.event_end_date|crmDate:0:1}{else}{$event.event_end_date|crmDate}{/if}{/if}
+       {event.event_title}<br />
```

Comment:
Should be `{event.title}`. The above comes out blank.

Reply:
The above comment is still true.

Reply:
This is still true ^^ but if you're doing a followup can do it there.

For the $totalAmount, yeah I'm not sure the text needs to be conditional. And the proposition one - the way that's used may not translate well in all languages so there's probably a better way to do it.